src/test/http-mocks/claim-store.ts

```diff
@@ -309,3 +310,15 @@ export function resolveRetrieveDocument () {
     .get(new RegExp('/.+/[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}'))
     .reply(HttpStatus.OK)
 }
+
+export function resolveUserRoles () {
+  mock(`${serviceBaseURL}/users`)
+    .get('/roles')
+    .reply(HttpStatus.OK, 'cmc-new-features-consent-given, citizen, admin, some-role')
```

Comment:
Please remove `citizen, admin, some-role` as it does not reflect real life scenario.